src/components/Location/Search.css

```diff
@@ -8,7 +8,6 @@
 }
 
 .search-icon {
-  width: var(--search-logo-width);
```

Comment:
I'm not sure but  maybe search-logo-width is just too small for some browsers for 1.2em font size and the search icon.